source/crud/crud.rst

```diff
@@ -202,7 +202,7 @@ Read
      * On servers >= 3.2, the default is no value: no 
      * "bypassDocumentValidation" option is sent with the "aggregate" command.
```

Comment:
The changes you made are fine, but they don't address the main point of SPEC-357, which was that lines 202-203 are ambiguous.

It sounds like it's saying "On servers >= 3.2 ... no bypassDocumentValidation option is sent with the aggregate command."

It's hard to interpret that sentence correctly.

The proposed clarification was to state clearly that on servers >= 3.2 the bypassDocumentValidation is sent to the server only if the application explicitly provided it.